### PR TITLE
fix pino to fail seilently

### PR DESCRIPTION
**Pull request by thecraftman**
- make the process fail silently; if an error occurs during this process (e.g., network issues), the catch block catches the error

fixes: https://linear.app/axiom/issue/AXM-2570/fix-logging-failures-in-axiomhq-pino-logger
- https://linear.app/axiom/issue/AXM-2571/revise-logging-libraries-to-avoid-application-crashes

### CODE REVIEW

File: packages/pino/src/index.ts [hunk 1]
```diff
@@ -17,21 +17,25 @@ export interface Options extends ClientOptions {
...
-
         const event = {
           _time: time,
           level: mapLogLevel(level),
           ...rest,
         };
 
-        axiom.ingest(dataset, event);
+        try {
+          await axiom.ingest(dataset, event);
```
Comment (c-ehrlich) [inline packages/pino/src/index.ts:33]: do we want to await here? seems like not blocking is probably better, and how it was done before?

Comment (thecraftman) [reply packages/pino/src/index.ts:33]: @c-ehrlich, if we remove the `await` and collect the promises returned by each ingestion? as it's now non-blocking

Comment (c-ehrlich) [reply packages/pino/src/index.ts:33]: @thecraftman 👍

Comment (thecraftman) [reply packages/pino/src/index.ts]: @c-ehrlich fixed

**Pull request closed without merging**